configs/centerpoint/centerpoint_pillar02_second_secfpn_8xb4-cyclic-20e_nus-3d.py

```diff
@@ -6,7 +6,7 @@
 
 # If point cloud range is changed, the models should also change their point
 # cloud range accordingly
-point_cloud_range = [-51.2, -51.2, -5.0, 51.2, 51.2, 3.0]
+point_cloud_range = [-51.2, -52, -5.0, 51.2, 50.3, 3.0]
```

Comment:
Does the 'point_cloud_range" in the file `centerpoint_voxel01_second_secfpn_8xb4-cyclic-20e_nus-3d` need to be modified?